portal/src/graphql/portal/CreateProjectScreen.tsx

```diff
@@ -106,10 +108,8 @@ function CreateProjectScreenContent(props: CreateProjectScreenContentProps) {
   );
 
   const onChangePhoneNumber = useCallback(
-    (_e, newValue) => {
-      if (newValue != null) {
-        setState((prev) => ({ ...prev, phoneNumber: newValue }));
-      }
+    (_e, phoneNumber: string, dialCode?: string) => {
```

Comment:
The `_e` is actually the E.164 phone number we need. Refer to PhoneScreen to see PhoneScreen stores both values for different purpose.